### ISSUE TITLE
Question: Local subdomain install

### ISSUE THREAD

**Issue by nobicycle** [mooradal/youtubeDownloader#1]
Hello mooradal,

I installed in an existing subdomain: joomla.localdomain. 

> $ cd /srv/http/joomla
> [http@tower joomla]$ cd Server/
> [http@tower Server]$ node index.js
> Server Works !!! At port 4000
>  
![e3e1e441-08b4-4f41-8f27-48ce303b63b1](https://user-images.githubusercontent.com/1372447/52763704-cd876000-3057-11e9-986a-e869e65a3a4b.png)

In script.js I tried localhost  and joomla.localdomain

> $ cat script.js 
> var convertBtn = document.querySelector('.convert-button');
> var URLinput = document.querySelector('.URL-input');
> 
> convertBtn.addEventListener('click', () => {
> 	console.log(`URL: ${URLinput.value}`);	
> 	sendURL(URLinput.value);
> });
> 
> function sendURL(URL) {
> 	window.location.href = `http://localhost:4000/download?URL=${URL}`;
> }

But I get in both cases :

> ERR_CONNECTION_REFUSED 

Will the example work in a subdomain?

Best

**Comment by NismoGraphics** [comment]
If it's still a problem, try using `window.location.href = window.location.href+"/download?URL="+URL;`

**Comment by nobicycle** [comment]
Thanks NismoGraphics.
That did not work so I decided to try it on an internet server.

> npm run dev
> 
> > server@1.0.0 dev /opt/youtubeDownloader/Server
> > nodemon index.js
> 
> [nodemon] 1.18.10
> [nodemon] to restart at any time, enter `rs`
> [nodemon] watching: *.*
> [nodemon] starting `node index.js`
> Server Works !!! At port 4000

I tried an nginx reverse proxy by disabling a monitor subdomain netdata.toraja.social, using it to test youtubeDownloader 

 cat /etc/nginx/sites-available/youtube 
upstream backend2 {
    server 127.0.0.1:4000;
#server unix:/tmp/netdata.sock;
    keepalive 64;
}

server {
    # nginx listens to this
    listen netdata.toraja.social:80;

    # the virtual host name of this
    server_name netdata.toraja.social;

    location / {
        proxy_set_header X-Forwarded-Host $host;
        proxy_set_header X-Forwarded-Server $host;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_pass http://backend2;
        proxy_http_version 1.1;
        proxy_pass_request_headers on;
        proxy_set_header Connection "keep-alive";
        proxy_store off;
    }
}

**Comment by rahulkumarsingh73690** [comment]
Yes, I don't Know about this Project But Node Youtube Dl Works Fine in Subdomain.
